### PR TITLE
add ontology_get_associated_samples

### PR DESCRIPTION
**Pull request by zhlu9890**
- [ ] I updated the README.md docs to reflect this change.

For changes to the codebase:

- [ ] I have written tests to cover this change.
- [x] This is not a breaking API change OR
- [ ] This is a breaking API change and I have incremented the API version and added a summary to CHANGELOG.md.

### CODE REVIEW

File: spec/stored_queries/ontology/ontology_get_associated_samples.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Get all samples reference this term
```
Comment (ialarmedalien) [inline spec/stored_queries/ontology/ontology_get_associated_samples.yaml:1]: Missing some words/letters in this line?

You can add a `description` parameter below `name` with this information in it.

File: spec/stored_queries/ontology/ontology_get_associated_samples.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+name: ontology_get_associated_samples
+params:
+  type: object
+  required: [id, ts, "@onto_terms"]
+  properties:
+    id:
+      type: string
+      title: Document ID
+      description: Ontology id of the term you want to get all the ancestors of
```
Comment (ialarmedalien) [inline spec/stored_queries/ontology/ontology_get_associated_samples.yaml:11]: capitalise ID

File: spec/stored_queries/ontology/ontology_get_associated_samples.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        description: Result offset for pagination
+        maximum: 100000
+    ts:
+      type: integer
+      title: Versioning timestamp
+    "@onto_terms":
+      type: string
+      title: Ontology terms collection name
+query_prefix: WITH samples_nodes
+query: |
```
Comment (ialarmedalien) [inline spec/stored_queries/ontology/ontology_get_associated_samples.yaml:29]: Can you add some automated tests for this query?

File: spec/stored_queries/ontology/ontology_get_associated_samples.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        type: integer
+        default: 0
+        description: Result offset for pagination
+        maximum: 100000
+    ts:
+      type: integer
+      title: Versioning timestamp
+    "@onto_terms":
+      type: string
+      title: Ontology terms collection name
```
Comment (ialarmedalien) [inline spec/stored_queries/ontology/ontology_get_associated_samples.yaml:27]: Can you explain this more and/or give some examples?

See example of a stored query with full descriptions:

https://github.com/kbase/relation_engine/blob/develop/spec/stored_queries/djornl/djornl_fetch_clusters.yaml

**Comment by jayrbolton** [comment]
The query looks okay to me. @zhlu9890 Do you think you would have time to add tests for this?

**Comment by zhlu9890** [comment]
@jayrbolton, I don't have time to add test for now.

**Review by ialarmedalien — CHANGES_REQUESTED** [review]
Please fix the capitalisation and clarify the description of `@onto_terms`.

**Comment by zhlu9890** [comment]
I have fixed the capitalisation and add the description and examples of @onto_terms.

**Review by ialarmedalien — APPROVED** [review]
👍

**Pull request merged**